lib/spack/spack/test/database.py

```diff
@@ -60,6 +61,8 @@ def upstream_and_downstream_db(tmpdir_factory, gen_mock_layout):
         downstream_db, downstream_layout
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
```

Comment:
It looks like all the DB tests are skipped: while IMO it's fine to skip upstream DB testing, I think we'll want to test the most basic DB functionality

Reply:
Agreed as per our discussion on slack.